browser_patches/chromium/build.sh

```diff
@@ -12,34 +12,53 @@ cd output
 
 FOLDER_NAME=""
 ZIP_NAME=""
-FILES_TO_REMOVE=()
+FFMPEG_URL=""
+FFMPEG_BIN_PATH=""
+FFMPEG_LICENSE_PATH=""
+CHROMIUM_FILES_TO_REMOVE=()
+
 
 if [[ $1 == "--win32" ]]; then
   FOLDER_NAME="Win"
   ZIP_NAME="chrome-win.zip"
-  FILES_TO_REMOVE+=("chrome-win/interactive_ui_tests.exe")
+  CHROMIUM_FILES_TO_REMOVE+=("chrome-win/interactive_ui_tests.exe")
+  FFMPEG_URL="https://playwright2.blob.core.windows.net/builds/ffmpeg/${FFMPEG_VERSION}/ffmpeg-win32.zip"
+  FFMPEG_BIN_PATH="ffmpeg-${FFMPEG_VERSION}-win32-static/bin/ffmpeg.exe"
+  FFMPEG_LICENSE_PATH="ffmpeg-${FFMPEG_VERSION}-win32-static/LICENSE.txt"
 elif [[ $1 == "--win64" ]]; then
   FOLDER_NAME="Win_x64"
   ZIP_NAME="chrome-win.zip"
-  FILES_TO_REMOVE+=("chrome-win/interactive_ui_tests.exe")
+  CHROMIUM_FILES_TO_REMOVE+=("chrome-win/interactive_ui_tests.exe")
+  FFMPEG_URL="https://playwright2.blob.core.windows.net/builds/ffmpeg/${FFMPEG_VERSION}/ffmpeg-win64.zip"
+  FFMPEG_BIN_PATH="ffmpeg-${FFMPEG_VERSION}-win64-static/bin/ffmpeg.exe"
+  FFMPEG_LICENSE_PATH="ffmpeg-${FFMPEG_VERSION}-win64-static/LICENSE.txt"
 elif [[ $1 == "--mac" ]]; then
   FOLDER_NAME="Mac"
   ZIP_NAME="chrome-mac.zip"
+  FFMPEG_URL="https://playwright2.blob.core.windows.net/builds/ffmpeg/${FFMPEG_VERSION}/ffmpeg-mac.zip"
+  FFMPEG_BIN_PATH="ffmpeg-${FFMPEG_VERSION}-macos64-static/bin/ffmpeg"
+  FFMPEG_LICENSE_PATH="ffmpeg-${FFMPEG_VERSION}-macos64-static/LICENSE.txt"
 elif [[ $1 == "--linux" ]]; then
   FOLDER_NAME="Linux_x64"
   ZIP_NAME="chrome-linux.zip"
+  FFMPEG_URL="https://playwright2.blob.core.windows.net/builds/ffmpeg/${FFMPEG_VERSION}/ffmpeg-linux.zip"
+  FFMPEG_BIN_PATH="ffmpeg-${FFMPEG_VERSION}-amd64-static/ffmpeg"
+  FFMPEG_LICENSE_PATH="ffmpeg-${FFMPEG_VERSION}-amd64-static/GPLv3.txt"
 else
   echo "ERROR: unknown platform to build: $1"
   exit 1
 fi
 
-URL="https://storage.googleapis.com/chromium-browser-snapshots/${FOLDER_NAME}/${UPSTREAM_CHROMIUM_REVISION}/${ZIP_NAME}"
-curl --output upstream.zip "${URL}"
-
-unzip upstream.zip
-
-for file in ${FILES_TO_REMOVE[@]}; do
+CHROMIUM_URL="https://storage.googleapis.com/chromium-browser-snapshots/${FOLDER_NAME}/${UPSTREAM_CHROMIUM_REVISION}/${ZIP_NAME}"
+curl --output chromium-upstream.zip "${CHROMIUM_URL}"
+unzip chromium-upstream.zip
+for file in ${CHROMIUM_FILES_TO_REMOVE[@]}; do
   rm -f "${file}"
 done
 
+curl --output ffmpeg-upstream.zip "${FFMPEG_URL}"
+unzip ffmpeg-upstream.zip
+cp "$FFMPEG_BIN_PATH" ${ZIP_NAME%.zip}
```

Comment:
Is adding ffmpeg to the binary also just a temporary implementation until we have native video recording available in Chromium?
Since static built ffmpeg is quite large ~75 MB - just thoughts.

Reply:
It'll be ~30MB since we only need `ffmpeg` binary.

> Is adding ffmpeg to the revisions also just a temporary implementation until we have native video recording available in Chromium?

It'll probably be medium-term to longterm.

Reply:
Strange, I downloaded [that](https://playwright2.blob.core.windows.net/builds/ffmpeg/4.3.1/ffmpeg-linux.zip) and inside the ffmpeg binary has 75 MB for me.

![image](https://user-images.githubusercontent.com/17984549/91896144-a96e8c80-ec98-11ea-8453-fe29e63a9626.png)

But nvm, go for it!